site/content/en/docs/Concepts/contexts.md

```diff
@@ -178,6 +178,27 @@ can still be interrupted if total demand for on demand instances in an availabil
 full details see [Spot Instance Interruptions](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-interruptions.html) 
 and [EC2 Spot Pricing](https://aws.amazon.com/ec2/spot/pricing/).
 
+### Ongoing Costs
+
+Until a context is destroyed resources that are deployed can incur ongoing costs even if a workflow is not running. The
+exact costs depend on the configuration of the context.
+
+Amazon Genomics CLI version 1.0.1 and earlier used an AWS Fargate based WES service for each deployed context. The service
+uses 0.5 vCPU, 4 GB memory and 20 GB base instance storage. Fargate pricing varies by region and is detailed [here](https://aws.amazon.com/fargate/pricing/).
+As at November 8, 2021, each WES service costs $0.03802 per hour in the us-east-1 region.
```

Comment:
Could we include a link to a calculator with the relevant components here using the aws calculator: https://calculator.aws/#/estimate

It would allow users to change the region easily as well if they didn't run their workloads in us-east-1

Reply:
+1 - this would keep our docs up to date even when AWS pricing changes as well

Reply:
Links added